RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -907,9 +907,9 @@ class DeepTauId : public deep_tau::DeepTauBase {
         get(dnn::neutralIsoPtSumdR03_over_dR05) = getValue(tau.tauID("neutralIsoPtSumdR03") / tau.tauID("neutralIsoPtSum"));
         get(dnn::photonPtSumOutsideSignalCone) = getValueNorm(tau.tauID("photonPtSumOutsideSignalConedR03"), 1.731f, 6.846f);
         get(dnn::puCorrPtSum) = getValueNorm(tau.tauID("puCorrPtSum"), 22.38f, 16.34f);
-        get(dnn::tau_dxy_pca_x) = getValueNorm(tau.dxy_PCA().x(), -0.0241f, 0.0074f);
-        get(dnn::tau_dxy_pca_y) = getValueNorm(tau.dxy_PCA().y(),0.0675f, 0.0128f);
-        get(dnn::tau_dxy_pca_z) = getValueNorm(tau.dxy_PCA().z(), 0.7973f, 3.456f);
+        get(dnn::tau_dxy_pca_x) = 0; // getValueNorm(tau.dxy_PCA().x(), -0.0241f, 0.0074f);
```

Comment:
Hi,
I will add a comment why zero is replacing the PCA point and why the original code is kept as set of commented lines. If it is not done I suppose such a comment from RECO. BTW, is it mandatory to keep original code knowing that usage of absolute PCA coordinates is meaningless?

Reply:
> BTW, is it mandatory to keep original code knowing that usage of absolute PCA coordinates is meaningless?

The commented part can be removed if you prefer. I kept it to keep track of what input is expected by the NN (i.e. which variable range was used during the training).

Reply:
So, let's keep it. But, please add a line or two explaining the reasons of zeroing and keeping commented code. Not big deal but useful, no?

Reply:
Well, in theory the neural net could still use PCA if we had fixed samples with comparable contents in data and MC (won't happen very soon of course...). So it's maybe not bad to keep the commented part to see what the net expects as Konstantin stated.
@mbluj please go ahead with adding a comment to the PRs (both for master and backport are set up). I guess it's a good idea to add it in advance. Will you also notify XPOG?

Reply:
Crossing messages... Do I understand correctly that you would add the comment in the code not the PR description? Would indeed be good to stay aware of the reason

Reply:
Yes, sure. I committed a comment that explains the reason of zeroing the PCA inputs: https://github.com/kandrosov/cmssw/commit/6c4e57e061c4aae9f48dbae3f987dc130a4f1a44 Please, let me know if it is ok.

Reply:
Yes, I mean to have comment also in code, I can add it if not done by Konstantin. I can notify XPOG, but I think it is better to to so when PR to 10_2_X is created. Do you want me create the set of PRs?
Concerning adding back the non-PV-relative PCA point to the NN - I am skeptical as it is not granted (or even expected) that position of BS is same in data and MC.

Reply:
The PRs already exist. I can add Konstantin's comment maybe also adding the note `Kept as comment to document what NN expects` and applying the code formatting request by jenkins. It wants two spaces in front of the comment